src/Athena/Content/css/site.css

```diff
@@ -37,18 +37,15 @@
     overflow-y: hidden;
 }
 
-    .tabs-content.carousel .carousel-item {
-        width: 100%;
-        height: 550px;
-    }
+.tabs-content.carousel .carousel-item {
+    width: 100%;
+    height: 550px;
+}
 
 html, body {
     height: 100%;
```

Comment:
Nuke this as well:

With `height: 100%`:

![image](https://user-images.githubusercontent.com/794251/37802313-586744f2-2e00-11e8-861f-c0f7dfa577b7.png)


Without `height: 100%`:

![image](https://user-images.githubusercontent.com/794251/37802297-47978204-2e00-11e8-8e04-aea3f11cf276.png)